spec/controllers/activity_forms_controller_spec.rb

```diff
@@ -39,6 +39,18 @@
         end
       end
 
+      context "ispf_theme step" do
+        subject { get_step :ispf_theme }
+
+        it { is_expected.to skip_to_next_step }
```

Comment:
I think it would be, but let's do it as a separate piece of work to keep this PR small. Would you mind adding a trello card to the technical debt column for this?

Reply:
Ah, thanks for clearing that up Cristina - you're right. Sorry @yndajas I should've looked a bit more closely at that implementation before asking you to write up that card

Reply:
Oh, very interesting! I did also wonder how it got the actual/activity, but it makes more sense if it's whatever action is being tested. Thanks!